mne/minimum_norm/resolution_matrix.py

```diff
@@ -201,25 +201,21 @@ def _summarise_psf_ctf(funcs, mode, n_comp, return_pca_vars):
     s_var = None  # only computed for return_pca_vars=True
 
     if mode == 'maxval':  # pick PSF/CTF with maximum absolute value
-
-        absvals = np.abs(funcs).max(axis=0)
+        absvals = np.array([abs(col).max() for col in funcs.T])
```

Comment:
You can use the same trick here
```suggestion
        absvals = np.maximum(-np.min(funcs, axis=0), np.max(funcs, axis=0))
```